source/use-cases/sensitive-data-encryption.txt

```diff
@@ -39,31 +39,33 @@ MedcoMD needs this system to satisfy the following use cases:
 
-The next MongoDB security feature they evaluated was `Role-Based Access Control <https://docs.mongodb.com/manual/core/authorization/>`_ which allows them to define distinct permissions for users. However, this solution did not allow them to differentiate access to specific fields within a document nor keep the data safe when transmitted over an insecure network.  
+The next MongoDB security features they evaluated were `Encryption at Rest <https://docs.mongodb.com/manual/core/security-encryption-at-rest/>`_ which encrypts the database files on disk and `Transport Encryption using TLS/SSL <https://docs.mongodb.com/manual/core/security-transport-encryption/>`_ which encrypts data over the network. When applied together, these two features prevent access to on-disk database files as well as capture of the data on the network, respectively. When combined with Role-Based Access Control, these three security features offer near-comprehensive security coverage of the sensitive data, but lack a mechanism to prevent the data from being read from the server's memory.
 
-Finally, the MedcoMD developers discovered Client-side Field Level Encryption which allows them to specify fields of a document that should be stored in encrypted form on the server. Before sending the data to the server, the client encrypts the specified, sensitive fields. After retrieving data from the server, the client decrypts any encrypted fields transparently and automatically. This prevents any unencrypted sensitive data from being passed over the network as well as access to the data by a user with direct access to the database.
+Finally, the MedcoMD developers discovered a feature that independently satisfies all the security criteria. Client-side Field Level Encryption allows the developers to specify the fields of a document that should be kept encrypted. Sensitive data is transparently encrypted/decrypted by the client and only communicated to and from the server in encrypted form. This mechanism keeps the specified data fields secure in encrypted form on both the server and the network. While all clients have access to the non-sensitive data fields, only appropriately-configured CSFLE clients are able to read and write the sensitive data fields.
 
-The Doctors will be provided with a client with CSFLE enabled and will therefore be able to access the sensitive data. The Receptionists will be provided a client without CSFLE enabled and therefore will never be able to access the sensitive data nor accidentally leave it displayed on-screen in a public area.
+The Receptionists are be provided a client without CSFLE enabled and therefore cannot view the sensitive data fields nor accidentally leave them displayed on-screen in a public area. Doctors are provided a client with CSFLE enabled and therefore be able to access the sensitive data fields in the privacy of their office. 
```

Comment:
nit: Let's add a transition sentence that goes from "CSFLE satisfies the requirements" to "now that they have CSFLE the developers are satisfied and have decided on a solution"

Reply:
Let's reframe using active voice:

MedcoMD will provide Receptionists with a client that is not configured to access data encrypted with CSFLE...